parastell/parastell.py

```diff
@@ -291,6 +292,112 @@ def export_source_mesh(self, filename='source_mesh', export_dir=''):
             export_dir=export_dir
         )
 
+    def construct_nwl_build(
+        self, toroidal_extent, wall_s, mat_tag='Vacuum', **kwargs
+    ):
+        """Construct special InVesselBuild class object for neutron wall loading
+        geometry.
+
+        Arguments:
+            toroidal_extent (float): angular extent to model in toroidal
+                direction [deg].
+            wall_s (float): closed flux surface label extrapolation at wall.
+            mat_tag (str): DAGMC material tag to use for neutron wall loading
+                geomery (defaults to 'Vacuum').
+
+        Optional attributes:
+            repeat (int): number of times to repeat build segment for full model
+                (defaults to 0).
+            num_ribs (int): total number of ribs over which to loft for each
+                build segment (defaults to 61). Ribs are set at toroidal angles
+                interpolated between those specified in 'toroidal_angles' if
+                this value is greater than the number of entries in
+                'toroidal_angles'.
+            num_rib_pts (int): total number of points defining each rib spline
+                (defaults to 61). Points are set at poloidal angles interpolated
+                between those specified in 'poloidal_angles' if this value is
+                greater than the number of entries in 'poloidal_angles'.
+            scale (float): a scaling factor between the units of VMEC and [cm]
+                (defaults to m2cm = 100).
+        """
+        self.radial_build = ivb.RadialBuild(
+            np.linspace(0.0, toroidal_extent, num = 4),
+            np.linspace(0.0, 360.0, num = 4),
+            wall_s,
+            radial_build={},
+            logger=self._logger,
+            **kwargs
+        )
+        
+        if wall_s > 1.0:
+            self.radial_build.radial_build['nwl_geom'] = \
+                self.radial_build.radial_build.pop('sol')
+            del self.radial_build.radial_build['plasma']
+        else:
+            self.radial_build.radial_build['nwl_geom'] = \
+                self.radial_build.radial_build.pop('plasma')
+        
+        self.radial_build._set_mat_tag('nwl_geom', mat_tag)
+
+        self.invessel_build = ivb.InVesselBuild(
+            self._vmec_obj,
+            self.radial_build,
+            logger=self._logger,
+            **kwargs
+        )
+
+        self.invessel_build.populate_surfaces()
+        self.invessel_build.calculate_loci()
+        self.invessel_build.generate_components()
+
+    def export_nwl_build(
```

Comment:
If we're able to streamline away from a special NWL build, can we also streamline away from a special NWL export?